### ISSUE TITLE
Test flake TestImageStreamMappingCreate

### ISSUE THREAD

**Issue by smarterclayton** [openshift/origin#5444]
```
--- FAIL: TestImageStreamMappingCreate-2 (6.23s)
	imagestream_test.go:215: unexpected error: imageStream "test" cannot be updated: the object has been modified; please apply your changes to the latest version and try again
FAIL
```

Likely due to image stream changes.

**Comment by deads2k** [comment]
Looks like a dupe of https://github.com/openshift/origin/issues/5421

**Comment by soltysh** [comment]
#5421 was resolved with #5455 please close.

**Comment by deads2k** [comment]
resolved with #5455
